api/models.py

```diff
@@ -37,28 +35,32 @@ class Meta:
             )
         ]
     )
-    priority_levels = postgresfields.ArrayField(models.CharField(max_length=8, choices=PRIORITY_CHOICES))
     title = models.CharField(null=True, max_length=255, blank=False) # ex "Classifier Search"
     description = models.CharField(null=True, max_length=2048, blank=False) # optional description
     default_timeout = models.IntegerField(default=600) # default timeout, in seconds
     max_attempts = models.IntegerField(default=1) # max number of times this job can attempt to run
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+class ValidateTaskPriority(object):
+    def __init__(self, base):
+        self.base = base
+
+    def __call__(self, value):
+        print('in validator')
```

Comment:
Don't quite get how this would be used yet, but should this be a call to a logger instead of print?